### ISSUE TITLE
RpmOptions.java:32-40: Repository options: create class...

### ISSUE THREAD

**Issue by 0pdd** [artipie/rpm-adapter#281]
The puzzle `275-12a5bede` from #275 has to be resolved: 

https://github.com/artipie/rpm-adapter/blob/f5d7c1c115a1bee78dcd6bb3472899ca45c4a544/src/main/java/com/artipie/rpm/RpmOptions.java#L32-L40

The puzzle was created by @olenagerasimova on 19-Jun-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8), @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) please, pay attention to this issue

**Comment by 0crat** [comment]
The job [#281](https://github.com/artipie/rpm-adapter/issues/281) assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/df4fa214-5d63-4481-a136-205508cec5f8); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@592e6706-44da-4634-a6e2-8ab630bff16f, thread: PQ-CT2E6TK9B
com.jcabi.xml.StrictXML[124] java.lang.IllegalArgumentException: 2 error(s) in XML document: -1:-1: cvc-pattern-valid: Value 'gh:Week 26' is not facet-valid with respect to pattern '[a-z]{2}:[A-Z0-9a-z.\-#/]+' for type 'milestoneID'.;-1:-1: cvc-complex-type.2.2: Element 'successor' must have no element [children], and the value must be valid.
```

[0.53.15](https://github.com/zerocracy/farm): CID: [d9343bd1-0c22-455e-b4b0-110a52112773](https://www.0crat.com/footprint/CT2E6TK9B/d9343bd1-0c22-455e-b4b0-110a52112773), Type: "Job milestoned"

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#286](https://github.com/artipie/rpm-adapter/issues/286), [#287](https://github.com/artipie/rpm-adapter/issues/287) are still not solved.

**Comment by 0pdd** [comment]
The puzzle `275-12a5bede` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
Job was finished in 18 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0pdd** [comment]
@0pdd the puzzle [#287](https://github.com/artipie/rpm-adapter/issues/287) is still not solved; solved: [#286](https://github.com/artipie/rpm-adapter/issues/286).

**Comment by 0pdd** [comment]
@0pdd all 2 puzzles are solved here: [#286](https://github.com/artipie/rpm-adapter/issues/286), [#287](https://github.com/artipie/rpm-adapter/issues/287).
